### ISSUE TITLE
WARN  n.liftweb.http.CoreRequestVarHandler - RequestVar net.liftmodules.ng.Angular$HeadRendered$_ was set but not read

### ISSUE THREAD

**Issue by alexmnyc** [joescii/lift-ng#11]
I keep getting these warnings:

WARN  n.liftweb.http.CoreRequestVarHandler - RequestVar net.liftmodules.ng.Angular$HeadRendered$_ was set but not read

I changed all "val's" to defs based on other ticket recommendations with no success. Still keep getting the same warnings. What else can be the cause of this warning? Thank you.

**Comment by joescii** [comment]
Yeah, I get those too. I understand in principle what the warning means, but I'm not sure why lift-ng in particular causes these. It's been on my list of things to figure out but I've just not found the time/priority for it. They're a nuisance but I've never experienced issues related to it. 

Joe

**Comment by 99sono** [comment]
likewise
